examples/training-artifacts/101/ch1/ecommerce/validation-overrides.xml

```diff
@@ -0,0 +1,4 @@
+<validation-overrides>
+    <allow until='2024-12-15'>indexing-change</allow>
```

Comment:
I didn't provide a validation-overrides because I thought this is a common problem people bump into and they'll "discover" how to do it.

But your change might just remove some unnecessary boilerplate, so +1. It's just that we might need to refresh these. Any better ideas than Slack reminders? :)